src/SFA.DAS.AANHub.Application/CalendarEvents/Commands/CreateCalendarEvent/CreateCalendarEventCommandValidator.cs

```diff
@@ -0,0 +1,240 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Common;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.CalendarEvents.Commands.CreateCalendarEvent;
+
+public class CreateCalendarEventCommandValidator : AbstractValidator<CreateCalendarEventCommand>
+{
+    public const string CalendarTypeIdMustNotBeEmpty = "calendarId must have a value";
+    public const string CalendarTypeIdMustBeValid = "calendarId must have a valid value";
+    public const string EventFormatMustNotBeEmpty = "eventFormat must have a valid value";
+    public const string StartDateMustNotBeEmpty = "startDate must have a valid value";
+    public const string StartDateMustBeInFuture = "startDate must be a future date";
+    public const string StartDateMustBeLessThanEndDate = "startDate must be less than or equal to endDate";
+    public const string EndDateMustNotBeEmpty = "endDate must have a valid value";
+    public const string EndDateMustBeInFuture = "endDate must be a future date";
+    public const string EndDateMustBeLessThanEndDate = "endDate must be greater than or equal to startDate";
```

Comment:
bad naming.  Also I don't think we need this validation at all, the 'StartDateMustBeLessThanEndDate' does the same thing.  It will just be showing the same basic error with 2 messages